### ISSUE TITLE
No colors in the process buffer

### ISSUE THREAD

**Issue by dgaw** [howl-editor/howl#492]
I've noticed that the build tools that display color in a normal terminal don't display color in Howl's process buffer. I believe ANSI colors are supported so I'm wondering what might be wrong.

Steps to reproduce:
1. <kbd>ctrl</kbd> <kbd>shift</kbd> r
2. ``ls --color=auto``

Expected result:
* a list of files and directories with color

Actual result:
* no colors 

Interestingly, when I type ``ls --color=always`` then the colors do work so I presume it's something to do with ``ls`` (and other tools) not recognizing the color capability. I do have ``TERM=xterm-256color`` and ``COLORTERM=truecolor`` env variables though.

Any ideas?

Edit: I'm on the current master (fdf0997).

**Comment by rokf** [comment]
Isn't that the default behavior of most command line tools? When they're piped somewhere (**not a tty**) then they do not output color, except if forced.

**Comment by dgaw** [comment]
I think you're right @rokf. However, Spacemacs for example manages to work around this and build tools display color in its compilation buffer. I'll have a look at how they do it.

**Comment by nilnor** [comment]
Yes, Howl manages ANSI color output if present, but most tools don't emit if unless they detect that they're in a terminal, which is why they have to be forced. To get around this you'd have to fake a terminal using a psuedo TTY, which entails some effort.

**Comment by dgaw** [comment]
Thanks for the answers. I've checked the tools I use and they can all be forced to use color so I'm putting this one on a back burner.

**Comment by shalabhc** [comment]
This is related to #218 - terminal emulation.

**Comment by nilnor** [comment]
Sorry for the very long silence here.

Howl development has been quiet on `master` for years, while work continued on the `next` branch — that branch has now been merged. Master has changed substantially since the 0.6 release: a full port to GTK 4, and a rewrite of the command line and interaction APIs, among a lot else.

We're closing out the old backlog, and feature requests are being closed for good rather than parked. Howl is developed at a pace that doesn't support an open wishlist, and leaving these sitting here indefinitely implies a roadmap that doesn't exist. That's not a comment on the merit of the idea — thank you for taking the time to write it up.

Please don't reopen this one. Bug reports against current master are still welcome.
